src/Lib/Managers/NobManager.vala

```diff
@@ -133,6 +133,7 @@ public class Akira.Lib.Managers.NobManager : Object {
     }
 
     private void update_nob_layer () {
+        nob_layer.set_visible (true);
```

Comment:
It might seem wrong, but the problem is that if I remove it from there when I multi select items the select effect does not appear. As a workaround I added the same line in the `on_update_select_effect` function, hope it's the right place to put the instruction.

Reply:
@giacomoalbe This line is causing issues in the Vector Tool, specifically PathEditMode. When creating paths, the nobs should not be shown.